core/arch/arm/plat-bcm/conf.mk

```diff
@@ -16,14 +15,22 @@ CFG_TZDRAM_SIZE ?= 0x01000000 # 16MB
 CFG_SHMEM_START ?= ($(CFG_TZDRAM_START) - $(CFG_SHMEM_SIZE))
 CFG_SHMEM_SIZE ?=  0x01000000 # 16MB
 CFG_TEE_RAM_VA_SIZE := 0x400000 # 4MB
-endif
 
 ifeq ($(platform-flavor-armv8),1)
 $(call force,CFG_WITH_ARM_TRUSTED_FW,y)
 $(call force,CFG_PM_STUBS,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 endif
 
+ifeq ($(PLATFORM_FLAVOR),ns3)
+$(call force,CFG_BCM_SOTP,y)
+$(call force,CFG_GPIO, y)
+$(call force,CFG_BCM_GPIO,y)
+$(call force,CFG_PL022,y)
+$(call force,CFG_BCM_HWRNG,y)
```

Comment:
`CFG_BCM_HWRNG` ???

Reply:
moved the driver enabling configs out of this patch